tests/suites/internalViewer/internalViewer.test.ts

```diff
@@ -0,0 +1,16 @@
+import {test, expect} from '@playwright/test';
+
+test.describe('Test InternalViewer', async () => {
+    test('Test internalViewer header link', async ({page}) => {
+        page.goto('');
+        const link = page.locator('header').locator('a').getByText('Internal Viewer');
```

Comment:
afaik role locators are preferable.
`page.getByRole('a', {name: 'Internal Viewer'})`
and maybe you should have separate page object for header

https://playwright.dev/docs/locators#locate-by-role